components/annot-age.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <annot-explanation :explanation="explanation"></annot-explanation>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "annotate_age",
+  data() {
+    return {
+      explanation: 'This is an explanation for how to annotate age',
```

Comment:
Let's consider consistent use of single vs double quotes, especially now re: a JS linter. Thus far I have been using double quotes for strings and alternating to single quotes when quoting something within double-quoted strings, e.g. tag attribute values.

Reply:
Hey, thanks for the comment. That's a good point. How do you feel about just merging the https://github.com/metaneuro/annotation_tool/pull/75 PR first and then applying all that? I can do it manually here but I think that'd be faster.

Reply:
That's fine.

Reply:
That's fine.

Reply:
OK, PR #75 is merged. I will adopt the airbnb style with single quote strings here now.

Reply:
Oh, yeah. Happy to go with that either, main goal was consistency with these commits. I went with single quotes because that's what the airbnb style guide happens to do (in case we adopt this) and because they look nicer to me. Let me know if you'd prefer double quotes for all JS strings and I'll change it.